apps/hash-graph/lib/graph/src/snapshot/ontology/batch.rs

```diff
@@ -0,0 +1,125 @@
+use async_trait::async_trait;
+use error_stack::{IntoReport, Result, ResultExt};
+use tokio_postgres::GenericClient;
+
+use crate::{
+    snapshot::{
+        ontology::{OntologyExternalMetadataRow, OntologyIdRow, OntologyOwnedMetadataRow},
+        WriteBatch,
+    },
+    store::{AsClient, InsertionError, PostgresStore},
+};
+
+pub enum OntologyTypeMetadataRowBatch {
+    Ids(Vec<OntologyIdRow>),
+    OwnedMetadata(Vec<OntologyOwnedMetadataRow>),
+    ExternalMetadata(Vec<OntologyExternalMetadataRow>),
+}
+
+#[async_trait]
+impl<C: AsClient> WriteBatch<C> for OntologyTypeMetadataRowBatch {
+    async fn begin(postgres_client: &PostgresStore<C>) -> Result<(), InsertionError> {
+        postgres_client
+            .as_client()
+            .client()
+            .simple_query(
+                r"
+                    CREATE TEMPORARY TABLE ontology_ids_tmp
+                        (LIKE ontology_ids INCLUDING ALL)
```

Comment:
Really awesome we can use `LIKE` here instead of manually creating the tables!

Reply:
Yeah, this is really helpful. I did this for almost all tables, where it's possible. I, however, merged a few tables like the entity links to avoid having duplicated ids stored in two tables. I don't know if this is actually faster, but at least it's easier to read (less boilerplate)